packages/pipeline-editor/src/RuntimesWidget.tsx

```diff
@@ -28,12 +28,31 @@ import React from 'react';
 
 import { PipelineService, RUNTIMES_NAMESPACE } from './PipelineService';
 
+const RUNTIMES_METADATA_CLASS = 'elyra-metadata-runtimes';
+
+const addTrailingSlash = (url: string): string => {
+  return url.endsWith('/') ? url : url + '/';
+};
+
+const getGithubURLFromAPI = (apiEndpoint: string): string => {
+  // For enterprise instances the api is located at <hostname>/api/
+  let baseURL = new URL(apiEndpoint).origin;
+
+  // For github.com the api endpoint is located at api.github.com
+  if (baseURL.includes('api.github.com')) {
```

Comment:
See above comment. An example for a GHE URL is `https://api.github.ibm.com`, which wouldn't be correctly derived by the logic I initially outlined in the issue.